pmlc/conversion/gpu/gpu_to_vulkan.cc

```diff
@@ -71,6 +80,9 @@ static constexpr const char *kBindBufferInteger32 = "bindBufferInteger32";
 static constexpr const char *kBindBufferInteger64 = "bindBufferInteger64";
 
 static constexpr const int kByteBits = 8;
+static constexpr const int kBufferCopyModeInit = 0;
```

Comment:
I think these should be an enum attribute (generated by tblgen), but @jbruestle and @flaub might have other thoughts.  (And this could be done later if we decide to do it; just add a TODO.)

Reply:
Fixed